### PR TITLE
Fix upload large file sizes bug

### LINKED ISSUE #687: 🐛 [Bug] Two Upload Success Messages are sent to the client on Image Upload (User Profile)
### Two Upload Success Messages are sent to the client on Image Upload (User Profile)
Two Upload Success Messages are sent to the client on Image Upload (User Profile). Maybe even to uploads take place.

### Steps to reproduce the behavior
1. Upload Userprofile
2. Wait for successful upload
3. See two upload success Messages
4. ...
5. Profit

### Expected behavior

Only one upload and one message should take place

### LINKED ISSUE #691: 🐛 [Bug] Avatar upload - handle large filesizes and error exceeding the limit
### Avatar upload - handle large filesizes and error exceeding the limit
Uploading Avatar images over a certain treshhold will result in an error.
The error massage is not good readable, we have to find a different solution for that.
And afterwards one can not upload an image again !!!

- [ ] Define Filesize limit (on backend)
- [ ] Handle File exceeding that limit
- [ ] Display proper error Message (toast?)
- [ ] Make sure functionality is not blocked once an error regarding filesize occured

### Steps to reproduce the behavior
```
3 MB
Bildschirmfoto 2018-03-19 um 10.23.37.png
File is too big (2.83MiB). Max filesize: 0.5MiB.
```

image

### Expected behavior

Should work ;-)

### Additional context

See https://github.com/Human-Connection/Human-Connection/pull/605#pullrequestreview-241640341

### PR DESCRIPTION
**Pull request by aonomike**
## 🍰 Pullrequest

- Increase the minimum accepted image size to 3 MB
- Display toast with error message if maximum size is exceeded
- Resolve double upload submit to the backend

Screen Shot 2019-05-28 at 11 46 10 PM

### Issues

- fixes #691
- fixes #687  

### Todo

- [ ] Force update the image component to display the initial image

### CODE REVIEW

**Comment by mattwr18** [comment]
this is great @aonomike, almost there! we just need to add tests to cover the new lines of code/functionality

File: webapp/components/Upload/index.vue [hunk 1]
```diff
@@ -44,6 +45,14 @@ export default {
       }
     },
   },
+  watch: {
+    error() {
+      let that = this
```
Comment (roschaefer) [inline webapp/components/Upload/index.vue]: Was this not solved with EcmaScript? :thinking: could you try out with `error: () => {` arrow functions?

Comment (roschaefer) [reply webapp/components/Upload/index.vue]: Or, what is `this ` actually here? Could you explain? :thinking:

**Comment by roschaefer** [comment]
@aonomike I'm excited to see this in action! Thank you so much!

awesome

File: webapp/components/Upload/index.vue (hunk 1 above)
Comment (mattwr18) [reply webapp/components/Upload/index.vue]: I tried it out and it seems to work, but it breaks the tests, so maybe it doesn't or maybe the tests need to be modified somehow... to be honest, we copied and pasted this from the `vue2-dropzone` demo...

**Pull request merged**